### ISSUE TITLE
Issue similar to #1537

### ISSUE THREAD

**Issue by francoisw** [Varying-Vagrant-Vagrants/VVV#1898]
## Expected Behavior
Creation of a new VM 

## Current Behavior
After creation, the VM and associated drivers are destroyed automatically

## Steps to Reproduce (for bugs)

1. Installed latest versions of Virtualbox, Vagrant and VVV
2. Installed Virtualhosts Updater Plugin
3. duplicated vvv-config.yaml to vvv-custom-yaml - no new sites created yet, just default VVV sites
4. vagrant up
5. also tried vagrant up --provision

Then:

```
==> default: Running triggers after up...
==> default: Forcing shutdown of VM...
==> default: Destroying VM and associated drives...
/Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/dsl.rb:90:in `method_missing': undefined method `name=' for #<VagrantPlugins::Triggers::DSL:0x0000000100e559b0> (NoMethodError)
	from /Users/franwess/vagrant-local/Vagrantfile:765:in `block (2 levels) in <top (required)>'
	from /Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/dsl.rb:11:in `instance_eval'
	from /Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/dsl.rb:11:in `fire!'
	from /Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/action/trigger.rb:52:in `block in fire_triggers'
	from /Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/action/trigger.rb:51:in `each'
	from /Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/action/trigger.rb:51:in `fire_triggers'
	from /Users/franwess/vagrant-local/.vagrant/plugins/gems/2.4.6/gems/vagrant-triggers-0.5.3/lib/vagrant-triggers/action/trigger.rb:18:in `call'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/action/warden.rb:50:in `call'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/action/builtin/before_trigger.rb:23:in `call'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/action/warden.rb:50:in `call'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/action/builder.rb:116:in `call'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/action/runner.rb:102:in `block in run'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/util/busy.rb:19:in `busy'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/action/runner.rb:102:in `run'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/machine.rb:238:in `action_raw'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/machine.rb:209:in `block in action'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/environment.rb:613:in `lock'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/machine.rb:195:in `call'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/machine.rb:195:in `action'
	from /opt/vagrant/embedded/gems/2.2.5/gems/vagrant-2.2.5/lib/vagrant/batch_action.rb:86:in `block (2 levels) in run'
```

https://gist.github.com/francoisw/0d1a6ec2b6aac94abce1c961b3eff6aa

## Your Environment

* VVV version:  Latest downloaded today v2.2.5
* VVV Git Branch:  Master (I think)
* Vagrant version:
* VM Provider name: 
* VM Provider version: VirtualBox 6.0.8
* Operating System and version: Mac OS 10.14.4

**Comment by tomjn** [comment]
@francoisw can you include a **full** copy of the provision? It looks like your gist only includes the core provisioner log, but none of the splash screen, utility provisioners, site provisioners, vagrant output, etc. So your inline code snippet has no context :( and it's impossible to diagnose what you did and where it went wrong :(

Copy paste the entire output from the terminal, including the commands, and put them in a gist

Also, as an aside, did you install the vagrant triggers plugin? If so, eliminate it, delete it, uninstall it, purge it with extreme predjudice

**Comment by tomjn** [comment]
Also did you follow the instructions that fixed the problem in #1537?:

https://github.com/Varying-Vagrant-Vagrants/VVV/issues/1537#issuecomment-393865469

**Comment by francoisw** [comment]
Hi @tomjn ,

Thank you for getting back to me! It seems that the steps in #1537 solved the issue - apologies for not picking that up in the log that I sent through.

Seems everything is good to go now.

Kind regards
Francois Wessels
